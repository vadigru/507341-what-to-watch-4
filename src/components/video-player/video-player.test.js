import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const mock = {
  posterUrl: `https://url.com/poster.jpg`,
  previewUrl: `https://url.com/preview/video.mp4`,
  autoplay: true,
  mute: true
};

it(`Should render VideoPlayer component`, () => {
  const {previewUrl, autoplay, mute} = mock;
  const tree = renderer
    .create(
        <VideoPlayer
          src={previewUrl}
          autoplay={autoplay}
          muted={mute}
        />, {
          createNodeMock: () => {
            return {};
          }
        }).toJSON();

  expect(tree).toMatchSnapshot();
});
