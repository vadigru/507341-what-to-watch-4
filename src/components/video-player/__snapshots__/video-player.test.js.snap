// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render VideoPlayer component 1`] = `
<video
  autoPlay={true}
  height="175"
  muted={true}
  onClick={[Function]}
  src="https://url.com/preview/video.mp4"
  width="100%"
/>
`;
